WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -38,14 +38,26 @@
                         android:layout_height="wrap_content"
                         android:layout_margin="@dimen/margin_large">
 
+                        <ProgressBar
+                            android:id="@+id/my_site_icon_progress"
+                            android:layout_width="@dimen/blavatar_sz"
+                            android:layout_height="@dimen/blavatar_sz"
+                            android:layout_centerVertical="true"
+                            android:layout_marginEnd="@dimen/margin_large"
+                            android:layout_marginRight="@dimen/margin_large"
+                            android:indeterminate="true"
+                            android:visibility="gone"
+                            android:gravity="center_vertical" />
+
                         <org.wordpress.android.widgets.WPNetworkImageView
```

Comment:
There's no touch feedback when tapping the site icon image.  We could add the `android:foreground="?attr/selectableItemBackgroundBorderless"` attribute to the `my_site_blavatar` `WPNetworkImageView` to help users know the icon is tappable. 

It's not part of these changes, but we could add the `android:background="?attr/selectableItemBackgroundBorderless"` attribute to [this `RelativeLayout`](https://github.com/wordpress-mobile/WordPress-Android/blob/64d5ff6cd93a475a3c54f4594503d3fe35e4d7dd/WordPress/src/main/res/layout/my_site_fragment.xml#L31) to help users realize the top card is tappable to go to their site.  I didn't even know it was possible until testing these changes!

Reply:
I've added the background to the `RelativeLayout`, however I'm having some troubles adding the foreground to the ImageView. 
The image view is not always clickable, since the user doesn't always have rights to edit the site icon. When I set the foreground in the xml, the foreground is set even for sites, for which the user can't edit the site icon. 
I thought I'd solve the issue by setting the foreground programatically, but I can't make it work:(. The drawable is never `borderless`. I have not sure why, but I decided it's such a minor issue that I'll leave it be.